binding-library/python/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure Functions Python Binding for Kafka
+This project provides Kafka binding for Azure Functions written in Python that makes it easy
+to consume Kafka messages from one topic.
+
+# Features
+- Reading from a Kafka topic
+
+# Getting Started
+
+## Installation via PyPI
+pip install azure-functions-kafka-binding
+
+## Installation via Git
+`git clone https://github.com/Azure/azure-functions-kafka-extension`
+
+`cd ./bindings_library/python/`
+
+`python setup.py install`
+
+## Minimum Requirements
+- Python 3.6 and above
```

Comment:
Python 3.7 is not currently supported so might be better to keep this at 3.6 for now.